clang-tools-extra/clang-tidy/modernize/UseTrailingReturnTypeCheck.cpp

```diff
@@ -113,7 +113,7 @@ struct UnqualNameVisitor : public RecursiveASTVisitor<UnqualNameVisitor> {
 };
 } // namespace
 
-constexpr llvm::StringLiteral Message =
+constexpr llvm::StringLiteral MessageFunction =
```

Comment:
Nit: I think the name was clearer before. I read this as "the function that does something with the message". 

Maybe `ErrorMessage` would be more accurate?

Reply:
Ok! I think this change would have been better to do in #135383, where it clearly shows that you need 2 different error messages.